### ISSUE TITLE
Quasar & Trakd

### ISSUE THREAD

**Issue by Stathistor** [scakemyer/quasar#45]
Hi and congrats on your work. I am facing an issue, I have read all relevant threads but it is not covered. 
The issue has to do with watched collection series (or movies) in Quasar. I have recently moved to Quasar and tried to link to Trakd. It worked fine, all my series are here, also the ones I have already watched.  By selecting watched / unwatched there is no actual change. The system works fine for the series I watch now, they disappear. Plus there is no choice of "marking season as watched" so that they will disappear if it not possible to be done automatically. 

Thanks.

**Comment by pmentis** [comment]
@Stathistor good point as I face the same problem. At this very moment Quasar is one of the most reliable video add-ons and one of my favorites. Great job on this. There are only two things that keep me from using it as my main video add-on.
1)	The first is the problem that you mentioned. “Watched status” in trakt, doesn’t sync with the “My Movies” (Tract Collection) folder in Quasar. Even worst I can’t mark them manually as “watched” because this job has to be done episode by episode…!
2)	Without being mandatory as the first problem I’d like an option in settings in order to stop the downloading of the torrent and the deleting of the downloaded file automatically (without the prompt dialog).  If I turn of the prompt dialog then it keep downloading the file and it doesn’t delete the downloaded file.
